config/initializers/simple_form_bootstrap.rb

```diff
@@ -83,6 +83,22 @@
     end
   end
 
+  config.wrappers :horizontal_form_condensed_with_documentation, tag: 'div', class: 'form-group', error_class: 'has-error' do |b|
```

Comment:
There is no difference between :horizontal_form_condensed_with_documentation and :horizontal_form_condensed. Remove  :horizontal_form_condensed_with_documentation.